src/azul/health.py

```diff
@@ -108,6 +108,17 @@ def queues(self):
                 }
         return response
 
+    @memoized_property
+    def failures(self):
+        dynamodb = boto3.resource('dynamodb')
+        table = dynamodb.Table(config.dynamo_failure_message_table_name)
+        bundle_key_condition = {'MessageType': {'AttributeValueList': ['bundle'], 'ComparisonOperator': 'EQ'}}
```

Comment:
These lines are overly too long. Can you wrap all the dict literals?

Reply:
Just wrap all the dict literals in `failures()`?